### PR TITLE
Delete me offset

### PR DESCRIPTION
**Pull request by FL3SH**
- added options to configure offset for relative game #67 
- lint check as part of the build command #64 

Thank you @brandoncc

### CODE REVIEW

**Comment by ThePrimeagen** [comment]
@FL3SH I didn't seem to get an email for this glorious change!

File: src/game/delete-round.ts [hunk 1]
```diff
@@ -14,6 +14,30 @@ export class DeleteRound extends Round {
...
+            offset = 3;
+        }
+
+        if (isDefindedUserOffset) {
+            offset = Number(
+                await game.nvim.getVar("vim_be_good_delete_me_offset"),
+            );
+            console.log("delete-round#getColumnOffset - userOffset ", offset);
+        } else {
+            offset = Math.floor(Math.random() * (40 - 5)) + 5;
```
Comment (brandoncc) [inline src/game/delete-round.ts:36]: @ThePrimeagen @FL3SH this line will overwrite the value set on line 27, won't it?

Comment (FL3SH) [reply src/game/delete-round.ts:36]: aaa, my bad, of course `offset = 3` will always be overwritten ;/
Tbh I personally like it way more with random offset event in 'noob' this is why I missed it.

Fixed in #70

**Comment by lpanebr** [comment]
>
> I love the random offset.


Would it make sense if the minimum offset (+5) changed depending on the
difficulty? Say the harder the bigger the minimum offset..?

**Comment by FL3SH** [comment]
I am not sure about that. Can you open an issue about that, so we can discuss it?

**Comment by lpanebr** [comment]
there it it: #72

**Pull request merged**